src/app/children/child-details/child-details.component.ts

```diff
@@ -28,7 +28,7 @@ import uniqid from 'uniqid';
 import {AlertService} from '../../alerts/alert.service';
 import {School} from '../../schools/school';
 import {ChildrenService} from '../children.service';
-import {HealthCheck} from '../HealthCheck';
+import {HealthCheck} from '../health-checkup/HealthCheck';
```

Comment:
for consistency the file name should better be health-check.ts.
Angular CLI and Webstorm manage to relate the camel case and hyphonated spellings automatically

Reply:
Changed the Filename and fixed the imports